src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -82,5 +86,25 @@ export const InlineEditExample = {
       demo: <InlineEditConfirm />,
       props: { EuiInlineEditText },
     },
+    {
+      title: 'Loading state',
+      text: (
+        <>
+          <p>
+            Setting the <EuiCode>isLoading</EuiCode> prop to true will add a
+            spinner to the input element in <EuiCode>editMode</EuiCode> and add
+            the loading state to the confirm and cancel input buttons.
+          </p>
+        </>
+      ),
+      source: [
+        {
+          type: GuideSectionTypes.JS,
```

Comment:
This needs to be TSX, since you're using typescript `:` annotations in the file 
```suggestion
          type: GuideSectionTypes.TSX,
```